backend/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { KubeFastifyInstance } from '../types';
+
+export const writeAdminLog = (fastify: KubeFastifyInstance, data: object): void => {
+  const normalizedPath = path.join(__dirname, '../../../logs/adminActivity.log');
+  try {
+    fs.appendFile(normalizedPath, `${JSON.stringify(data)}\n`, function (err) {
+      if (err) {
+        return new Error(err.message);
```

Comment:
I suspected this didn't work...

```
[Error: ENOENT: no such file or directory, open '/Users/andrewballantyne/code/others/odh-dashboard/logs/adminActivity.log'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/andrewballantyne/code/others/odh-dashboard/logs/adminActivity.log'
}
```

Please verify the changes work before requesting a review. 

Also I think you want to throw an error, not return it. I don't think the callback for an async function will ever do anything as it is hard to fit it into a thread that the caller can use.